docs/concepts/dicom-cast.md

```diff
@@ -35,7 +35,7 @@ The current implementation of DICOM Cast has the following mappings:
 
 | Property | Tag Id | Tag Name | Required Tag?| Note |
 | :------- | :----- | :------- | :----- | :----- |
-| Patient.identifier.where(system = '') | (0010,0020) | PatientID | Yes | For now, the system will be empty string. We will add support later for allowing the system to be specified. |
+| Patient.identifier.where(system = 'system') | (0010,0020) | PatientID | Yes | Patient system id will set to the value of patientSystemId environment variable or Issuer of Patient Id dicom tag (0010, 0021) based on the isIssuerIdUsed boolean env variable. An empty string will be set by default if the variables are not defined. |
```

Comment:
nit: refer to the "env variables" as settings or configurations instead. The use of environment variables in the dockerfiles is an implementation detail

Reply:
Done